index.html

```diff
@@ -3103,7 +3105,7 @@ <h3>Production</h3>
           <li>
 Other Data Types not otherewise defined in the <a href="#data-model">data model</a> MUST be
 represented as a <a href="https://tools.ietf.org/html/rfc7049#section-2.1">CBOR
-String type. (major type 5)</a>
+String type</a> (major type 3).
```

Comment:
Mmm. I think this should be type 2 (byte string ) to preserve data.  Moving it to type 2 makes a transformation from potentially native binary format.

Reply:
Now that I think about it, isn't that whole sentence unnecessary? The table defines how to represent all allowed DID document data types in CBOR, and there simply are no other data types in the data model....

Reply:
@jonnycrunch can we remove this sentence, or can you explain its purpose? What is an example of a "data type not otherwise defined in the data model"?